misc/toml-parser/src/main/java/io/ballerina/toml/semantic/ast/TomlTransformer.java

```diff
@@ -74,7 +77,9 @@ public TomlNode transform(DocumentNode documentNode) {
     }
 
     private TomlTableNode createRootTable(DocumentNode modulePartNode) {
-        TomlKeyNode tomlKeyNode = new TomlKeyNode(null);
+        TomlKeyEntryNode root = new TomlKeyEntryNode(new TomlUnquotedKeyNode("__root", getPosition(modulePartNode)));
+        List<TomlKeyEntryNode> tomlKeyEntryNodes = Collections.singletonList(root);
+        TomlKeyNode tomlKeyNode = new TomlKeyNode(tomlKeyEntryNodes, getLocationOfKeyEntryList(tomlKeyEntryNodes));
         return new TomlTableNode(tomlKeyNode, getPosition(modulePartNode));
```

Comment:
`getPosition(modulePartNode)` is repeated. 
Also in L82, wouldn't the location is the same as `getPosition(modulePartNode)`? Do we need to calculate it again with `getLocationOfKeyEntryList()`?